mk/config.mk

```diff
@@ -306,6 +306,9 @@ $(eval $(call cfg-depends-all,CFG_REE_FS_TA_BUFFERED,CFG_REE_FS_TA))
 # in-tree TAs. CFG_IN_TREE_EARLY_TAS is formatted as:
 # <name-of-ta>/<uuid>
 # for instance avb/023f8f1a-292a-432b-8fc4-de8471358067
+#
+# By default the early TAs are compressed in the TEE binary, it is possible to
+# not compress them with CFG_EARLY_TA_COMPRESS=n
```

Comment:
Minor: I'd rather move this comment a few lines down, just above the line `CFG_EARLY_TA_COMPRESS ?= y`.